models/credmark/algorithms/minimum_risk.py

```diff
@@ -0,0 +1,63 @@
+from credmark.cmf.model import Model
+from credmark.dto import EmptyInput
+
+from models.credmark.protocols.lending.aave.aave_v2 import AaveDebtInfos
+from models.credmark.protocols.lending.compound.compound_v2 import CompoundV2PoolInfos
+
+
+@Model.describe(slug="finance.min-risk-rate",
+                version="1.0",
+                display_name="Calculate minimal risk rate",
+                description='Rates from stablecoins\' loans to Aave and Compound, '
+                            'then weighted by their debt size and total supply',
+                input=EmptyInput,
+                output=dict)
+class Minrisk(Model):
+    """
+    Doc is
+        https://docs.credmark.com/smart-money-in-defi/investment-concepts/minimum-risk-rate-of-defi
+    """
+    STABLECOINS = {
```

Comment:
we'll need a model to get the "stablecoins on a chain"

Reply:
ok. I added token.stablecoins.

Reply:
yay!